docs/manual.rst

```diff
@@ -2007,30 +2007,66 @@ For example, here is an identity function applicable to both types of input
 
     g2 = transform(id_func, g1)
 
-A partially applied transform function from pyproj satisfies the requirements
-for `func`.
+
+If using `pyproj>=2.1.0`, the preferred method to project geometries is:
 
 .. code-block:: python
 
+    import pyproj
+    import pyproj.crs
+
+    from shapely.geometry import Point
     from shapely.ops import transform
+
+    wgs84_pt = Point(-72.2495, 43.886)
+
+    wgs84 = pyproj.crs.CRS('epsg:4326')
+    utm = pyproj.crs.CRS('epsg:32618')
```

Comment:
```suggestion
    wgs84 = pyproj.CRS('epsg:4326')
    utm = pyproj.CRS('epsg:32618')
```

(unlike the API docs indicate, the CRS object is available top-level, and the preferred way to use it IMO)

Reply:
I missed that, thanks for the heads up!